pages/runtime/runtime.md

```diff
@@ -139,11 +139,20 @@ Alternatively, it is possible to reboot the device via the dbus interface as des
 
 ### Dbus communication with hostOS
 
-In some cases its necessary to communicate with the hostOS systemd to perform actions on the host, for example changing the hostname. To do this you can use [dbus][dbus-link]. In order to ensure that you are communicating to the hostOS systemd and not the systemd in your container it is important to set `DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host_run/dbus/system_bus_socket` for all dbus communication. Below you can find a couple of examples.
+In some cases its necessary to communicate with the hostOS systemd to perform actions on the host, for example changing the hostname. To do this you can use [dbus][dbus-link]. In order to ensure that you are communicating to the hostOS systemd and not the systemd in your container it is important to set `DBUS_SYSTEM_BUS_ADDRESS` for all dbus communication. The setting of that environment variable is different for resinOS 1.x and 2.x devices, choose the line that is correct for your device's OS version (can be found in your device dashboard):
+
+```
+# for resinOS 1.x use this version:
+DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host_run/dbus/system_bus_socket
```

Comment:
Yeah, seems to work since Supervisor 1.7.0, will update it accordingly
https://github.com/resin-io/resin-supervisor/commit/3d082aaaa8749b7dd9d274a61f71411e4b12467f